res/Sectors/M1105/Nooq.txt

```diff
@@ -0,0 +1,451 @@
+# Generated by https://travellermap.com
+# 2018-11-05T13:12:51+00:00
+
+# Nooq
+# 7,2
+
+# Name: Nooq (Hv)
+# Name: Hkimbiipam (Kk)
+
+# Milieu: M1105
+
+# Credits: Nooq sector was designed by Mark Humphreys.
+
+# Author:    Mark Humphreys
+
+# Subsector A: Iakkhe
+# Subsector B: 
+# Subsector C: 
+# Subsector D: 
+# Subsector E: Uakoia
+# Subsector F: 
+# Subsector G: 
+# Subsector H: 
+# Subsector I: Urnace
+# Subsector J: Regawrat
+# Subsector K: 
+# Subsector L: 
+# Subsector M: Amnega
+# Subsector N: 
+# Subsector O: 
+# Subsector P: 
+
+# Alleg: Ax: "Axanatglar Free State"
+# Alleg: Ux: "Uxiteck Trading Alliance"
+# Alleg: CsHv: "Hiver Client State"
```

Comment:
To be consistent with other sectors, this allegiance name should be "Client State, Hive Federation"

Reply:
Done